docs/content/preview/yugabyte-platform/install-yugabyte-platform/prerequisites.md

```diff
@@ -50,7 +50,9 @@ You prepare the host as follows:
 
   Replicated installs a compatible Docker version if it is not pre-installed on the host. The currently supported Docker version is 20.10.n.
 
-- For a Kubernetes-based installation, you need to ensure that the host can pull container images from the [Quay.io](https://quay.io/) container registry. For details, see [Pull and push YugabyteDB Docker images to private container registry](#pull-and-push-yugabytedb-docker-images-to-private-container-registry).
+- For a Kubernetes-based installation, you need to ensure that the host can pull container images from the [Quay.io](https://quay.io/) container registry. For details, see [Pull and push YugabyteDB Docker images to private container registry](#pull-and-push-yugabytedb-docker-images-to-private-container-registry). 
```

Comment:
@lizayugabyte : do you think we can create tabs on this page for Replicated vs Kubernetes, like we have on other pages like https://docs.yugabyte.com/preview/yugabyte-platform/install-yugabyte-platform/install-software/kubernetes/

That will allow us to specify separate pre-reqs for kubernetes clearly.

Reply:
@iSignal @bhavin192 This would require creating separate .md files for different installation types. Links to this section would be broken (and there are quite a few of them, including from external docs such as blogs). 
I restructured the "Prepare the host" section in such a way that installation types appear in the right-side menu. Please take a look and let me know if that's acceptable for now.

Reply:
@iSignal Yes, I agree, restructuring of prerequisites/prepare content has been on my "future improvements" list. I suggest merging the current PR to make the core dumps information public, and then I'll create a new doc ticket to address restructuring (I'll copy your input to this new ticket). Would that work?

Reply:
Yes, that makes sense, let's get this checked in for now.